vlasiator.cpp

```diff
@@ -494,17 +496,23 @@ int main(int argn,char* args[]) {
 
    phiprof::start("getFieldsFromFsGrid");
    volGrid.updateGhostCells();
-   getFieldsFromFsGrid(volGrid, BgBGrid, EGradPeGrid, technicalGrid, mpiGrid, cells);
+   getFieldsFromFsGrid(volGrid, BgBGrid, EGradPeGrid, dMomentsGrid, technicalGrid, mpiGrid, cells);
    phiprof::stop("getFieldsFromFsGrid");
 
+   phiprof::start("compute-dt");
+   // Run Vlasov solver once with zero dt to initialize
+   // per-cell dt limits and moments.
+   calculateSpatialTranslation(mpiGrid,0.0);
+   calculateAcceleration(mpiGrid,0.0);
+   /* 
    if (P::isRestart == false) {
-      phiprof::start("compute-dt");
       // Run Vlasov solver once with zero dt to initialize
       // per-cell dt limits. In restarts, we read the dt from file.
       calculateSpatialTranslation(mpiGrid,0.0);
       calculateAcceleration(mpiGrid,0.0);      
-      phiprof::stop("compute-dt");
    }
+   */
+   phiprof::stop("compute-dt");
```

Comment:
What is the confusion here? Is only the if removed and this shows weirdly?

Reply:
It *was* late, I oversaw the /* and the */ even though I noticed the grey text. Sorry.